presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix IS DISTINCT FROM for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve read-path to better estimate the size of row. Output (sum) string stat even if min/max values are too long.
+* Improve performance of array_join.
+* Improve efficiency of ST_Buffer. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances. https://prestodb.io/docs/current/functions/geospatial.html.
```

Comment:
Can we use `:ref:` instead of hard coded link here?

Reply:
:doc:`/functions/geospatial` is that more appropriate